src/PagedList/PagedList.cs

```diff
@@ -26,37 +26,16 @@ public class PagedList<T> : BasePagedList<T>
 		/// <param name="pageSize">The maximum size of any individual subset.</param>
 		/// <exception cref="ArgumentOutOfRangeException">The specified index cannot be less than zero.</exception>
 		/// <exception cref="ArgumentOutOfRangeException">The specified page size cannot be less than one.</exception>
-		public PagedList(IQueryable<T> superset, int pageNumber, int pageSize)
-		{
-			if (pageNumber < 1)
-				throw new ArgumentOutOfRangeException("pageNumber", pageNumber, "PageNumber cannot be below 1.");
-			if (pageSize < 1)
-				throw new ArgumentOutOfRangeException("pageSize", pageSize, "PageSize cannot be less than 1.");
-
-			// set source to blank list if superset is null to prevent exceptions
-			TotalItemCount = superset == null ? 0 : superset.Count();
```

Comment:
This is the only line that is different in the constructor code of PagedList and BasePagedList classes, so I put a base class constructor call here, and passed this expression for TotalItemCount.